include/boost/histogram/utility/clopper_pearson_interval.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 Jay Gohil, Hans Dembinski
+//
+// Distributed under the Boost Software License, version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_HISTOGRAM_UTILITY_CLOPPER_PEARSON_INTERVAL_HPP
+#define BOOST_HISTOGRAM_UTILITY_CLOPPER_PEARSON_INTERVAL_HPP
+
+#include <boost/histogram/utility/binomial_proportion_interval.hpp>
+#include <cmath>
+#include <utility>
+
+namespace boost {
+namespace histogram {
+namespace utility {
+
+template <class ValueType>
+class clopper_pearson_interval : public binomial_proportion_interval<ValueType> {
+  using base_t = binomial_proportion_interval<ValueType>;
+
+public:
+  using value_type = typename base_t::value_type;
+  using interval_type = typename base_t::interval_type;
+
+  explicit clopper_pearson_interval(deviation d = deviation{1.0}) noexcept
+      : z_{static_cast<double>(d)} {}
+
+  interval_type operator()(value_type successes, value_type failures) const noexcept {
+    const value_type half{0.5}, one{1.0};
+    const value_type ns = successes, nf = failures;
+    const value_type n = ns + nf;
+    // Source: https://en.wikipedia.org/wiki/Binomial_proportion_confidence_interval
+    const value_type alpha1 = (one - d) * half;
+    const value_type alpha2 = one - ((one - d) * half);
```

Comment:
This part looks wrong, there is only one alpha.

Reply:
And you cannot compute alpha from d, also there is no variable d in this scope here?

Reply:
I should rename it, it's 1st arguments for low and high variables (α/2 and 1-α/2). And yes I interchanged d with cl, so I'll α from cl/2 as you mentioned!